cypress/e2e/clue/branch/student_tests/graph_table_integraton_test_spec.js

```diff
@@ -15,6 +15,10 @@ const textToolTile = new TextToolTile;
 const x = ['3', '7', '6', '0'];
 const y = ['2.5', '5', '1', '0'];
 
+//TODO: All instances of "Geometry 1" were originally "Graph 1" and changed to pass test.
```

Comment:
Met with @emcelroy and he decided this was the best approach to pass the tests and will address them when CODAP is fully imported.

Reply:
FWIW: I _think_ I have a fix for this in [this PR](https://github.com/concord-consortium/collaborative-learning/pull/1726). Note the change in src/components/tiles/table/use-consumer-tile-linking.ts to use `titleBase` instead of `type` to construct a title in the case that the linkable tile has no title value. I'm going to ask Kirk to review those changes. If they get merged in before this PR does, I'll let you know.

I still don't 100% understand why this test seems to have only recently started to fail, though. I suspect recent changes to how tiles do or don't set a default title value clashed with how the table linking dialog creates a title value for linkable tiles that don't have a default title. If that's not true, then the only other thing I can think of is that the test should have been failing after we made [changes to table linking a couple weeks ago](https://github.com/concord-consortium/collaborative-learning/pull/1696), but wasn't for some oddball reason.

Reply:
Thanks @emcelroy . Let me know when that PR gets merged in.